pom.xml

```diff
@@ -1219,6 +1219,7 @@
     <activemqVersion>5.10.0</activemqVersion>
     <atomikosVersion>3.9.2</atomikosVersion>
     <camelVersion>2.14.1</camelVersion>
+    <elasticsearch-version>1.0.0_4</elasticsearch-version>
```

Comment:
Good catch! I probably meant to use it [here](https://github.com/OpenNMS/opennms/pull/336/files#diff-0eb738dd4425240b8e667d41580adf31R76) but missed it